example/vite.config.ts

```diff
@@ -20,6 +21,7 @@ export default defineConfig({
     }),
     serverAdapter({
       adapter,
+      getLoadContext,
```

Comment:
Also, I'm not 💯 in my testing, but when I added

```
    remixCloudflareDevProxy({
      getLoadContext,
    }),
```
and removed `getLoadContext` from `serverAdapter`, the example app still passed my bindings and augmented context through

Reply:
About `remixCloudflareDevProxy`, perhaps it may not need it because this `hono-remix-adapter` does the same thing. I'll test it; if so, I will create the PR.